docs/resourse-cleanup.md

```diff
@@ -0,0 +1,32 @@
+# Stale Resource Cleanup
+
+If the PVC is created with storage class which is having the reclaimPolicy
+as Retain will not delete the PV object and backend omap metadata and backend image.
+Manual deletion of PV will result in stale omap keys, values and also stale cephfs and rbd images.
+You need to cleanup metadata and image separately.
+
+### RBD
+
+**Find related image of pvc**:
+
+```[$]kubectl get pvc pvc_name -n namespace -oyaml``` to get pv_name <br>
+```[$]kubectl get pv pv_name -oyaml``` to get part of rbd_image_name(the last part of spec.csi.volumeHandle, xxx for example) <br>
+```[$]rbd ls -p pool_name | grep xxx``` to get full rbd_image_name
+
+**Delete metadata and image**:
```

Comment:
need to provide an option to delete omap data from rados
steps would be
* Get volume ID from PV
* Get omap keys
* Delete omap Keys
* Delete rbd image /Delete cephfs image
* Delete PV

providing an example would be a good one

Reply:
Does 'get omap keys' mean using 'block_name_prefix' of rbd_info to get all objects , then remove omapkeys of object one by one ?

Reply:
https://github.com/ceph/ceph-csi/blob/master/troubleshooting/tools/tracevol.py will help you to understand more on this

Reply:
I update my patch and thanks for your review.